### ISSUE TITLE
Expo 50 with  version 1.0.0-beta.15 crash startup app on iOS

### ISSUE THREAD

**Issue by AChevallier** [customerio/customerio-expo-plugin#129]
Hello!

I successfully built with the new  "customerio-expo-plugin": "1.0.0-beta.15".

```
"customerio-expo-plugin": "1.0.0-beta.15",
"customerio-reactnative": "3.4.0",
```
But my app is crashing as soon as start up the app on iOS.

`MessagingPushAPN.application(_:didFailToRegisterForRemoteNotificationsWithError:) + 68
2   libdispatch.dylib             	       0x18010d244 _dispatch_call_block_and_release + 24
3   libdispatch.dylib             	       0x18010ea98 _dispatch_client_callout + 16`

![Screenshot 2024-02-22 at 17 23 38](https://github.com/customerio/customerio-expo-plugin/assets/10862109/884bfd12-2293-437e-bb6a-2689092d8aeb)

Can you help me ?

**Comment by mrehan27** [comment]
Thank you for reaching out and sharing this with us. Sorry to hear about the crash you're experiencing, it definitely shouldn't be happening. To help us debug this issue further, can you please provide the following information?

- Are you working with managed or bare workflow in Expo?
- Can you share your `app.json` file?
- If possible, can you provide complete stack trace so we can gather more details?

Please make sure you remove any private information before sharing, such as keys and sensitive data. Or you can email us at win@customer.io, and include a link to this ticket for reference.

Please don't hesitate to ask any additional questions you may have.

**Comment by mrehan27** [comment]
Thank you for providing the details. We've received them and will continue to help you through email. I'm closing this issue here, but rest assured, our conversation and support will continue via email.

**Comment by vandemberg** [comment]
Hi, I'm experiencing the same issue with the same versions (Expo SDK 50.0.6). Do we have a default resolution for this problem?

**Comment by mrehan27** [comment]
Hi @vandemberg. The bug identified in this issue was fixed in React Native SDK version `3.5.2`. Can you please make sure you are using the latest SDK versions for both our Expo and React Native SDKs? If yes and the problem still persists, can you please share crash logs that you see so we can look into it and help you better? Thanks!
